S7.Net/PlcAsynchronous.cs

```diff
@@ -279,6 +269,20 @@ public async Task<List<DataItem>> ReadMultipleVarsAsync(List<DataItem> dataItems
             return dataItems;
         }
 
+        private static byte[] BuildReadRequestPackage(IList<DataRequestItem> dataItems)
+        {                
+            // first create the header
+            int packageSize = 19 + (dataItems.Count * 12);
+            var package = new System.IO.MemoryStream(packageSize);
+            BuildHeaderPackage(package, dataItems.Count);
+            // package.Add(0x02);  // datenart
```

Comment:
Not sure why this comment is here, also I really dislike German (or any non-English) comments in code.

Reply:
Yeah I could have cleaned that up when copy-pasting. will do.

Reply:
Didn't notice it was copy-paste, obviously you're not to blame for what someone else wrote.